PyeonHae/PyeonHae/Views/DetailItemView/DetailItemView.swift

```diff
@@ -9,12 +9,29 @@ import SwiftUI
 
 struct DetailItemView: View {
     
+    @State private var isReviewButtonVisible = false
+    
     var body: some View {
         ZStack {
             VStack(alignment: .leading) {
                 DetailItemViewTopBar
                 ScrollView {
                     ItemDetailView()
+                        .background(
+                            GeometryReader { geometry -> Color in
+                                let maxY = geometry.frame(in: .global).midY
+                                if maxY <= 0 && !isReviewButtonVisible {
+                                    DispatchQueue.main.async {
+                                        isReviewButtonVisible = true
+                                    }
+                                } else if maxY > 0 && isReviewButtonVisible {
+                                    DispatchQueue.main.async {
+                                        isReviewButtonVisible = false
+                                    }
+                                }
```

Comment:
조건문의 isReviewButtonVisible가 불 필요해 보이고

if ~ else if 로 나누지 않는게 코드가 더 깔끔해질 것 같습니다.

```suggestion
                              DispatchQueue.main.async {
                                    isReviewButtonVisible = maxY <= 0
                                }
```

Reply:
https://github.com/cvs-go/cvs-go-ios/pull/28/commits/b5fd917e02ff02efdd79d58794acff3b1c90f8b2

반영하였습니다! 감사합니다.